darity.go

```diff
@@ -26,6 +26,16 @@ const (
 // Constants taken from from <linux/kvm.h>, so cgo is not necessary.
 const (
 	kvmGetAPIVersion = 44544
+	kvmCreateVM      = 44545
+)
+
+type MachineType int
+
+const (
```

Comment:
A comment on this block would be great too, potentially explaining where these numbers are found.